deploy/helm-chart/values.yaml

```diff
@@ -100,3 +106,6 @@ metrics:
   ## Enable serving metrics via https. By default metrics are served via http.
   ## Please note that this endpoint will be secured with a self-signed certificate.
   secure: false
+
+# extraVolumes for the NGINX Gateway Fabric pod. Use in conjunction with nginxGateway.extraVolumeMounts and nginx.extraVolumeMounts to mount additional volumes to the containers.
```

Comment:
The linter is complaining about this line length. Can you split the comment into two lines?